### PR TITLE
Add a helpfile for ICC

### PR DESCRIPTION
**Pull request by jansim**
Title says it all, as the ICC analysis is currently still missing a proper helpfile, this PR adds one.

### CODE REVIEW

File: inst/help/intraclassCorrelation.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- between 0.75 and 0.90: good
+- above 0.90: excellent
+
+### References
+-------
+- Field, A. P., Miles, J., & Field, Z. (2012). Discovering statistics using R. Sage.
+- Revelle, W. (2021) psych: Procedures for Personality and Psychological Research, Northwestern University, Evanston, Illinois, USA, https://CRAN.R-project.org/package=psych Version = 2.1.9.
+- Shrout, P. E., & Fleiss, J. L. (1979). Intraclass Correlations: Uses in Assessing Rater Reliability. 9.
+- Cicchetti, D. V. (1994). Guidelines, criteria, and rules of thumb for evaluating normed and standardized assessment instruments in psychology. Psychological assessment, 6(4), 284.
+- Koo, T. K., & Li, M. Y. (2016). A guideline of selecting and reporting intraclass correlation coefficients for reliability research. Journal of chiropractic medicine, 15(2), 155-163.
```
Comment (vandenman) [inline inst/help/intraclassCorrelation.md:46]: I'm aware it's pedantic, but could you use markdown syntax to properly format these according to APA? So *italic* by typing \*italic\*.

**Review by vandenman — CHANGES_REQUESTED** [review]
Looks good! One small remark, see below.

In general, feel free to assign a reviewer because otherwise, nobody knows that you opened a PR.

**Comment by vandenman** [comment]
When are the ICCs supposed to be available in JASP again?

File: inst/help/intraclassCorrelation.md (hunk 1 above)
Comment (jansim) [reply inst/help/intraclassCorrelation.md]: Haha, fair enough.. first time ever doing this by hand after years of Zotero doing all the work for me 😄

**Comment by jansim** [comment]
iirc ICCs are already available?

**Review by vandenman — APPROVED** [review]

**Pull request merged**